src/Symfony/Component/Messenger/MessageBusInterface.php

```diff
@@ -27,5 +27,5 @@ interface MessageBusInterface
      *
      * @throws ExceptionInterface
      */
-    public function dispatch(object $message, array $stamps = []): Envelope;
+    public function dispatch(object $message, array $stamps = []): Envelope|array;
```

Comment:
AFAIK this is BC break.

Reply:
Yes, I need to change it. Maybe I should add a new interface

Reply:
Yes, I updated it